config.toml

```diff
@@ -1,8 +1,8 @@
 [database]
   host     = "localhost"
   port     = 5432
-  dbname   = "tagger"
-  user     = "user"
+  dbname   = "tagger-dev"
```

Comment:
should this be `tagger-db`?

Reply:
I meant it more that this is the development database, but it doesn't matter. Changed.